### PR TITLE
Implemented News API to pull articles to Web Servlet.

### PR DESCRIPTION
**Pull request by trinitydonohugh**

### CODE REVIEW

File: src/main/java/com/google/sps/servlets/articlesServlet.java [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+@WebServlet("/articleLinks")
+public class articlesServlet extends HttpServlet {
+  private static final String baseURL = "https://newsapi.org/v2/everything?domains=healthline.com&apiKey=";
+  private static final Logger logger = Logger.getLogger(articlesServlet.class.getName());
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    StringBuilder strBuf = new StringBuilder();  
+    
+    HttpURLConnection conn = null;
```
Comment (mbouadeus) [inline src/main/java/com/google/sps/servlets/articlesServlet.java:53]: Implementation looks good but is there no google maven dependency to facilitate calling the News API?

File: src/main/java/com/google/sps/data/Article.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  public Article(ArrayList<String> source, String author, String title, 
+    String description, URL url, String publishedAt) {
+    this.source = source;
+    this.author = author;
+    this.title = title;
+    this.description = description;
+    this.url = url;
+    this.publishedAt = publishedAt;
+  }
+}
```
Comment (mbouadeus) [inline src/main/java/com/google/sps/data/Article.java:39]: no new line at EOF.

**Review by mbouadeus — APPROVED** [review]
Overall, LGTM

**Review by isaacwdavis — APPROVED** [review]

File: src/main/java/com/google/sps/servlets/articlesServlet.java (hunk 1 above)
Comment (trinitydonohugh) [reply src/main/java/com/google/sps/servlets/articlesServlet.java:53]: The News API is a public API, not a Google API, so I don't think there's a maven dependency for it :)

**Pull request merged**